scripts/select_ids_from_smithsonian.py

```diff
@@ -0,0 +1,56 @@
+import click
```

Comment:
This was used to generate some unit test output files (https://github.com/isamplesorg/metadata/pull/56).  If it belongs somewhere else, let me know!